### ISSUE TITLE
Run through 18F accessibility checklist

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#288]
There is a [simple checklist](https://accessibility.18f.gov/checklist/) provided by 18F that may give a "quick" Revised Section 508 compliance test result. Run through the list, document the results using [ARCT tooling](https://github.com/Section508Coordinators/ACRT), [VPAT Rev 508 document](https://www.itic.org/policy/accessibility/vpat), [spreadsheet](https://www.section508.gov/create/spreadsheets/authoring-guides/), or similar. If any fails seem fixable at this point, attempt to fix them.

Alternative to 18F checklist could be  [WCAG-EM Report Tool](https://www.w3.org/WAI/eval/report-tool/), in addition to or replacement of if 18F turns out to be unsatisfactory

Tooling that may help:

- [ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) chrome plugin
- [Accessibility Insights](https://accessibilityinsights.io/downloads/) chrome plugin
- [Color Contrast Analyzer](https://www.tpgi.com/color-contrast-checker/)
- [Web Developer](https://chrome.google.com/webstore/detail/web-developer/bfbameneiokkgbdmiekhjnmfkcnldhhm) extension
